components/eam/tools/topo_tool/cube_to_target/cube_to_target.F90

```diff
@@ -966,6 +1066,12 @@ subroutine usage()
    print *, '                                      3km grid (SGH30) is also downscaled,   '
    print *, '                                      but does not depend on the smoothing.  '
    print *, '                                                                             '
+   print *, '  --add-oro-shape                     Enable the calculate of orographic     '
```

Comment:
Typo here, should be "Enable the calculation of"

Reply:
fixed.